### PR TITLE
trim extra spaces and words

### PR DESCRIPTION
**Pull request by liliVicky**
Signed-off-by: liwenqi <vikilwq@zju.edu.cn>

### CODE REVIEW

File: contributors/design-proposals/container-runtime-interface-v1.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ react to changes in lifecycle.
 The only transitions that are possible for a container are described below:
 
 ```
-() -> Created        // A container can only transition to created from the
+() -> Created        // A container can only transition to create from the
```
Comment (jonboulle) [inline contributors/design-proposals/container-runtime-interface-v1.md]: This should not be changed.

Comment (liliVicky) [inline contributors/design-proposals/container-runtime-interface-v1.md]: I agree with you, but now how should I do?  Shoud I correct and pull request again?

Comment (jonboulle) [inline contributors/design-proposals/container-runtime-interface-v1.md]: Add a new commit reverting this part, then squash into the previous one.

Comment (liliVicky) [inline contributors/design-proposals/container-runtime-interface-v1.md]: Ok, thank you!

Comment (liliVicky) [inline contributors/design-proposals/container-runtime-interface-v1.md]: I have modified. @jonboulle

**Comment by jonboulle** [comment]
lgtm, but when you squash, you can collapse the commit messages together

**Comment by liliVicky** [comment]
Thank you! @jonboulle

**Pull request merged**